pkgs/unified_analytics/USAGE_GUIDE.md

```diff
@@ -120,6 +120,19 @@ final Analytics analytics = Analytics(...);
 print('This user's status: ${analytics.telemetryEnabled}');  // true if opted-in
 ```
 
+## Developing Within `package:unified_analytics`
+
+When contributing to this package and the developer needs to verify that
+events have been sent, the following credentials should be used in
+`lib/src/constants.dart` to point to a separate Google Analytics endpoint
+
+```dart
+const String kGoogleAnalyticsApiSecret = '4yT8__oER3Cd84dtx6r-_A';
```

Comment:
Can you verify that this is ok to commit to a repo?

Reply:
I believe so, we have been committing the secrets for the legacy analytics package in flutter_tools because it has to be included for each POST request to GA